changelogs/fragments/added_uri_tests.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - new tests url, uri and urn will verify string as such, but they don't check existance of the resource
```

Very likely you won't need a changelog fragment for new test/filter plugins anymore, thanks to sidecar docs, but if you do, it should look as follows:
```suggestion
add plugin.test:
  - name: uri
    description: check if string is a valid URI
  - name: url
    description: check if string is a valid URL
  - name: urn
    description: check if string is a valid URN
```

---

Actually no changelog fragment is needed anymore (just tried that), see also #77871.